src/ml_board/frontend/Dashboard/src/components/dashboard/Dashboard.tsx

```diff
@@ -49,94 +49,41 @@ export default function Dashboard() {
-      // color: colorMap[experiment.experiment_id],
-      // ...scores
-    });
+    rows.push(row);
   }
-  console.log(rows.at(-1));
-
-
-
-  // // loop over all job ids
-  // const job_ids = useAppSelector(selectJobIds);
-  // for (const jID of job_ids) {
-  //   // get job and the experiment
-  //   const job: Job = useAppSelector(state => selectJobById(state, jID)) as Job;
-  //   const expID = job.experiment_id as string;
-  //   const experiment: Experiment = useAppSelector(state => selectExperimentById(state, expID)) as Experiment;
-
-  //   // calc the progresses here for better code readablity 
-  //   let epoch_progress = -1;
-  //   if (experiment.current_epoch !== undefined && experiment.num_epochs !== undefined) {
-  //     epoch_progress = experiment.current_epoch / experiment.num_epochs;
-  //   }
-  //   let batch_progress = -1;
-  //   if (experiment.current_batch !== undefined && experiment.num_batches !== undefined) {
-  //     epoch_progress = experiment.current_batch / experiment.num_batches;
-  //   }
-
-  //   // // if the scores/"latest_split_metric" needs handling before showing them in the table? (I don't think so)
-  //   // const scores = {
-  //   //   F1: experiment.F1 as string,
-  //   //   Precision: experiment.Precision as string,
-  //   //   Recall: experiment.Recall as string,
-  //   // }
-
-  //   // create the TableRow and push it
-  //   rows.push({
-  //     ...job,
-  //     ...experiment,
-  //     epoch_progress: epoch_progress,
-  //     batch_progress: batch_progress,
-  //     color: colorMap[experiment.experiment_id],
-  //     // ...scores
-  //   });
-  // }
-
-
-  // REMOVED because the type 'undefined' is not a valid JSX element
-  // // TODO: this might be redundant???
-  // const iActiveTab = useAppSelector(selectTab);
-  // if ("Dashboard" !== iActiveTab)
-  //   return;
+  // console.log(rows.at(-1));
 
   // TODO: get the colNames from the redux state ???
   // IMPORTANT NOTE: these names have to match the keys of the row object exactly in order to appear in the table
   const colNames = ["job_id", "job_type", "job_status", "experiment_id", "starting_time", "finishing_time",
     "epoch_progress", "batch_progress", "model_status", "current_split", "splits", "error",
-    "stacktrace", "device", "current_epoch", "num_epochs", "F1", "Precision", "Recall"];
+    "stacktrace", "device", "current_epoch", "num_epochs", "train_F1_SCORE_macro", "train_PRECISION_macro",
+    "train_RECALL_macro", "train_cross_entropy_loss", "val_F1_SCORE_macro", "val_PRECISION_macro", "val_RECALL_macro",
+    "val_cross_entropy_loss", "test_F1_SCORE_macro", "test_PRECISION_macro", "test_RECALL_macro", "test_cross_entropy_loss"];
 
-  return (
-    <div>
-      Dashboard
-      <Table colNames={colNames.filter((colName: string) => re.test(colName))} rows={rows} />
-    </div>
-  );
+  return (<Table colNames={colNames.filter((colName: string) => re.test(colName))} rows={rows} />);
```

Comment:
![image](https://user-images.githubusercontent.com/48678390/220759686-971e7268-0e83-4c13-a4af-6f827c50a783.png)

On scrolling the table, the table's header row also get's scrolled. As you can seen in the image.
Ideally, shouldn't it remain fixed and we can scroll other rows ?